docs/index.md

```diff
@@ -852,6 +854,14 @@ Mocha supports the `err.expected` and `err.actual` properties of any thrown `Ass
 
 ![string diffs](images/reporter-string-diffs.png?withoutEnlargement&resize=920,9999){:class="screenshot" loading="lazy"}
 
+### Max diff size
+
+By default strings are truncated to 2048 characters before generating a diff. This is to prevent performance problems with large strings.
+
+It can however make the output harder to interpret when comparing large strings. Therefore it is possible to configure this value to suit different needs, using `--reporter-option maxDiffSize=[number]`.
+
+A value of 0 indicates no limit, default is 2048 characters.
+
```

Comment:
Done!

Reply:
I forgot to add `> _New in v9.2.1_` before your description, sorry.
I'm going to classify this PR as a patch fix, since it's not really a new feature. If you agree.

8092 or 8192?

Reply:
8192 of course, patch will do just fine :)

Reply:
Can you add `> _New in v9.2.1_`, please.

Reply:
Done